service/src/test/java/bio/terra/workspace/common/BaseAwsConnectedTest.java

```diff
@@ -8,8 +8,8 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ActiveProfiles;
 
-/** Base class for AWS tests. Treat these as connected tests: connected to AWS */
-@Tag("aws")
+/** Base class for AWS connected tests: connected to AWS */
+@Tag("aws-connected")
```

Comment:
I think it's better not to include this tag in `BaseAwsConnectedTest` and continue annotating each test class directly. If we follow the `connectedPlus` pattern for pre- vs. post-submit tests from other clouds, having this here would tag tests with both `aws-connected` and `aws-connectedPlus`

Reply:
Makes sense. changed